sriovnet.go

```diff
@@ -24,7 +24,11 @@ const (
 	ibEncapType    = "infiniband"
 )
 
-var virtFnRe = regexp.MustCompile(`virtfn(\d+)`)
+var (
+	virtFnRe     = regexp.MustCompile(`virtfn(\d+)`)
+	auxDeviceRe  = regexp.MustCompile(`(\s+)\.(\s+).(\d+)`)
```

Comment:
not sure how this regex works. 
1. \s matches any whitespace character (equivalent to [\r\n\t\f\v ])
2.  you are missing \ before the second dot. otherwise it wildcard 
3. you don't use the grouping ()
I think it should be \S+\.\S+\.\d+